gensrc/proto/olap_file.proto

```diff
@@ -146,6 +146,8 @@ message RowsetMetaPB {
     optional bool enable_inverted_index_file_info = 1006;
     repeated InvertedIndexFileInfo inverted_index_file_info = 1007;
     optional SchemaDictKeyList schema_dict_key_list = 1008; // align to cloud rowset
+    // the time when this rowset should become stale (for compaction input rowsets)
+    optional int64 stale_at = 56;
```

Comment:
is this necessary? we wont use stale rowset after BE restarts
it seems it is OK we keep it in memory.
just a discussion, not a suggestion

Reply:
the less we touch the metadata, the more stable the application becomes.